main.py

```diff
@@ -110,7 +115,8 @@ def main(cluster_name, regions, sleep, kubeconfig, pod_namespace,
                       notifier=notifier,
                       dry_run=dry_run,
                       drainable_labels=drainable_labels,
-                      scale_label=scale_label
+                      scale_label=scale_label,
+                      instance_type_priorities=instance_type_priorities
```

Comment:
I think it's because it comes into play when `sorted` is used, and that handles strings too.